Docs/Configuration.tex

```diff
@@ -1543,16 +1527,16 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
 
   \begin{itemize}
     \tightlist
-    \item CSM region can be marked as boot services code or data, which
+    \item The CSM region can be marked as boot services code, or data, which
       leaves it as free memory for XNU kernel.
-    \item MMIO regions can be marked as reserved memory and stay unmapped,
-      but may be required to be accessible at runtime for NVRAM support.
+    \item MMIO regions can be marked as reserved memory and stay unmapped.
+      They may however be required to be accessible at runtime for NVRAM support.
   \end{itemize}
 
-  This quirk attempts to fix types of these regions, e.g. ACPI NVS for
+  This quirk attempts to fix various types of these regions, e.g. ACPI NVS for
   CSM or MMIO for MMIO.
 
-  \emph{Note}: The necessity of this quirk is determined by artifacts, sleep
+  \emph{Note}: The need for this quirk is determined by artifacts, sleep
   wake issues, and boot failures. Only very old firmware typically need
```

Comment:
need -> needs (change needed in original), firmware is singular

Reply:
Not clear. 
Please expand on what you mean.

Reply:
Apologies: "Only very old firmware typically **needs**"

Reply:
> "Only very old firmware typically needs"

Ambivalent on that. Best to disambiguate the context.
`This quirk is typically only required by very old firmware.`